Skip 7.0.x failing tests

jamesfredley opened the pull request:
Adds `@PendingFeature` annotation to run but skip failing tests, adds `@Ignore` when test cannot be run at all without stopping build this adds 3 skipped tests to the ~15 that existed previously  https://github.com/grails/grails-core/pull/13702/files

re-enable tests in gradle.yml for snapshots https://github.com/grails/grails-core/pull/13702/commits/48575024a4c576ecd773f27c335b443ff0c45cc2

Before https://github.com/grails/grails-testing-support/pull/435
![image](https://github.com/user-attachments/assets/07d995ea-2e72-4b92-a8f2-f89df4aec106)

After https://github.com/grails/grails-testing-support/pull/435
![image](https://github.com/user-attachments/assets/31f66736-e3f1-485b-aa1d-04774cd016bd)

jamesfredley committed:
Fix 3 ResponseMethodSpec tests that were failing

jamesfredley committed:
remove @PendingFeature since this test now passes

jamesfredley committed:
Add @PendingFeature annotation to run but skip failing tests, add @Ignore when test cannot be run at all

jamesfredley committed:
re-enable tests in gradle.yml for snapshots

jamesfredley commented:
Run with commit prior to Micronaut removal.  The total includes some tests which get rerun.  

![image](https://github.com/user-attachments/assets/5c2a269e-1746-4a02-a7ed-179cfc7f97cc)

codeconsole commented:
I started down that route, but adding that many skips seemed a bit extreme. 

Especially considering something like this takes out most of them 
https://github.com/grails/grails-testing-support/pull/431#event-14443183319

jamesfredley committed:
Remove @Ignore from 22 test which pass after https://github.com/grails/grails-testing-support/pull/435
These tests require StaticMessageSource instead of PluginAwareResourceBundleMessageSource

jamesfredley committed:
Restore original contentType for 3 tests

jamesfredley committed:
Remove @PendingFeature from tests which pass after https://github.com/grails/grails-testing-support/pull/435

jamesfredley committed:
Add @PendingFeature to test that was failing previously

jamesfredley committed:
cleanup imports

jamesfredley committed:
cleanup

jamesfredley committed:
Remove @Ignore from passing tests
